modules/apps/frontend-theme/kyoung-liferay-theme/src/js/canvas.js

```diff
@@ -0,0 +1,89 @@
+var space;
+
+function floatySpace() {
+    var colors = [
+        '#FF3F8E', '#04C2C9', '#2E55C1'
+    ];
+
+    space = new CanvasSpace('canvas', '#252934').display();
+    var form = new Form(space);
+
+	// Elements
+
+    var pts = [];
+    var center = space.size.$divide(1.8);
+    var angle = -(window.innerWidth * 0.5);
+	var count = window.innerWidth * 0.05;

```

Comment:
Since we use `window.innerWidth` a few times, we should cache this as a JS variable to save time looking up the property on the `window` object.
Ex.
```
var innerWidth = window.innerWidth;

var angle = -(innerWidth * 0.5);
var count = innerWidth * 0.05;
```

Reply:
Since you are doing the same logic twice (`innerWidth * 0.5`), save the time and create a JS variable for that and reuse it in both JS statements.

Reply:
Child JS statements should be indented 1 tab/4 spaces, but if they are not children statements, they should be at the same level as the statement above.
Ex.
```
var a = ...;
var b = ...;

if (a) {
    b;
    
    var c = ...;

    if (c) {
        ...
    }
}
```